amazon-qldb-driver/Cargo.toml

```diff
@@ -8,12 +8,10 @@ edition = "2018"
 
 [dependencies]
 amazon-qldb-driver-core = { version = "*", path = "../amazon-qldb-driver-core" }
-amazon-qldb-driver-rusoto = { version = "*", path = "../amazon-qldb-driver-rusoto" }
-tokio = "1.12.0"
-rusoto_core = { version = "0.47.0", default_features = false, features = ["rustls"] }
-rusoto_qldb_session = { version = "0.47.0", default_features = false, features = ["rustls"] }
+tokio = "1.11.0"
```

Comment:
`tokio` could be `1.12.0` instead? 
https://github.com/awslabs/amazon-qldb-driver-rust/commit/aa00db3640eab45364c199d93e4f25eb0da91c6a#diff-99ebf4a33a9b6967d3f845096b09b1f0b991150a400bd411cb80126f6a1156f6

Reply:
Fixed!